src/core_modules/capture-core/components/WorkingLists/EventWorkingListsCommon/useDefaultColumnConfiguration/useDefaultColumnConfig.js

```diff
@@ -21,6 +21,12 @@ const getDefaultMainConfig = (stage: ProgramStage): Array<MainColumnConfig> => {
         // $FlowFixMe[prop-missing] automated comment
         type: elementTypeKeys.DATE,
         header: stage.stageForm.getLabel(mainPropertyNames.OCCURRED_AT),
+    }, {
+        id: mainPropertyNames.ORGANISATION_UNIT,
+        visible: true,
+        type: elementTypeKeys.ORGANISATION_UNIT,
+        header: i18n.t('Organisation unit'),
+        apiName: 'orgUnit',
```

Comment:
In Events Programs, when sorting by orgUnit and saving/updating the WL the `orgUnitId` is stored in the `order` field. The other WLs look good now.

https://github.com/user-attachments/assets/461945ad-91da-4827-b97a-de7c461586ae

Could you have a look? Thank you!